gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/ChannelPool.java

```diff
@@ -84,7 +84,7 @@ static ChannelPool create(ChannelPoolSettings settings, ChannelFactory channelFa
       throws IOException {
 
     // ProcessHandle was introduced in Java 9
-    System.out.println("ProcessHandle: " + java.lang.ProcessHandle.current());
+    System.out.println("org.graalvm.home.Version: " + org.graalvm.home.Version.parse("1.2.3"));
```

Comment:
When using `openjdk version "17.0.6" 2023-01-17`, the Maven test `sdk-platform-java$ mvn test -am -pl gax-java/gax-grpc` (without the jvm property) succeeds.

Reply:
The Java 8 check https://github.com/googleapis/sdk-platform-java/actions/runs/4982584325/jobs/8918462230?pr=1681 successfully detected this gap:

```
Error:  Errors: 
Error:    ChannelPoolTest.callShouldCompleteAfterCreation:254 » UnsupportedClassVersion org/graalvm/home/Version has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
Error:    ChannelPoolTest.callShouldCompleteAfterStarted:303 » UnsupportedClassVersion org/graalvm/home/Version has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
```